src/overlays/actors/ovl_En_Elfgrp/z_en_elfgrp.c

```diff
@@ -27,54 +45,541 @@ const ActorInit En_Elfgrp_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void func_80A396B0(EnElfgrp* this, s32 arg1) {
+    for (; arg1 > 0; arg1--) {
+        if (this->actor.cutscene == -1) {
+            break;
+        }
+        this->actor.cutscene = ActorCutscene_GetAdditionalCutscene(this->actor.cutscene);
+    }
```

Comment:
```suggestion
    while (arg1 > 0) {
        if (this->actor.cutscene == -1) {
            break;
        }
        this->actor.cutscene = ActorCutscene_GetAdditionalCutscene(this->actor.cutscene);
        
        arg1--;
    }
```
Would this still match?